src/utils.js

```diff
@@ -238,3 +237,58 @@ export function parseCspPolicy(policy) {
 
   return parsedPolicy;
 }
+
+/**
+ * Determines if the source text is minified.
+ * Using the percentage no. of the indented lines from a sample set of lines
+ * to determine if the js file is minified.
+ * Inspired by code for the Firefox Developer Toolbar.
+ */
+export function couldBeMinifiedCode(code) {
+  // Number of lines to look at, taken from the head of the code.
+  const sampleSize = 30;
+
+  // Threshold in percent of indented lines to mark a file as not
+  // minified.
+  const indentCountThreshold = 20; // percentage
+
+  // Lenght of a line that looks suspicious of being minified
+  const hugeLinesLength = 500;
+
+  // Number of huge lines to also mark a file as potentially minified
+  // Hint: Minified AngularJS has 12 lines, jQuery 4
+  const hugeLinesThreshold = 4;
```

Comment:
I would consider reducing this to 1? I think it's quite unlikely to have a sane un-minified src file with over 500chars on a single line - or do you have a reason to think that would lead to false-positives?

Reply:
I could think of base64 encoded files, people do that for absolutely no reason 🤷‍♂️

Reply:
And, the indentation detection would detect files that are minified but have <4 lines that are super long so I'm not too worried.